src/style.css

```diff
@@ -1,3 +1,93 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
 body {
-  background-color: bisque;
-}
\ No newline at end of file
+  margin: 0;
+  padding: 0;
+
+  /* reserved for add styling feature
+  font-family: 'Mulish', Helvetica, sans-serif, Arial;
+  color: #fff;
+  background-color: #2d3838;
+  */
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.